cmds/modules/noded/main.go

```diff
@@ -202,6 +205,41 @@ func action(cli *cli.Context) error {
 		return errors.Wrap(err, "failed to create a new perfMon")
 	}
 
+	if exists := iperf.Exists(zinit.Default()); exists {
```

Comment:
As discussed before running a tests against public nodes happens from ALL nodes (even the ones without public config). The idea is:
- Public nodes running iperf running in test mode 
- All nodes run test against the nodes that has iperf server running 
- Nodes measure their up/down speed against the random public nodes.